CMakeLists.txt

```diff
@@ -128,43 +128,14 @@ endif()
 ###############################################################################
 ### FMS
 ###############################################################################
-include(fms_files.cmake)
```

Comment:
Remove `fms_files.cmake` if it's not used anymore

Reply:
fms_files.cmake has been deleted